src/utils/react/PropsToOmitUtils.ts

```diff
@@ -0,0 +1,8 @@
+const internalComponentPropsToOmit: string[] = ['children', 'dangerouslySetInnerHTML'];
+
+const inputTagPropsToOmit: string[] = [...internalComponentPropsToOmit, 'defaultValue', 'onClick', 'onChange', 'onBlur', 'value', 'valid'];

```

Comment:
I think I would add a small comment about why those props are to omit something in the line of `We omit those props because when the input was created we didn't use the correct types.`